Result.xcodeproj/project.pbxproj

```diff
@@ -191,7 +191,8 @@
 				D454806E1A9572F5009D7229 /* ResultTests.swift */,
 				D454806C1A9572F5009D7229 /* Supporting Files */,
 			);
-			path = ResultTests;
+			name = ResultTests;
```

Comment:
I’m a little surprised we don’t have to do the same thing with the sources as well.